src/__test__/components.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'; // eslint-disable-line
+import { render } from '@testing-library/react'; // eslint-disable-line
+import '@testing-library/jest-dom/extend-expect'; // eslint-disable-line
+import Calculator from '../components/calculator'; // eslint-disable-line
+import Home from '../pages/home'; // eslint-disable-line
+import Quote from '../pages/quote'; // eslint-disable-line
+import renderer from 'react-test-renderer'; // eslint-disable-line
+import App from '../App'; // eslint-disable-line
+
+describe('Test snapshot', () => {
+  test('Render Home', () => {
+    const component = render(<Home />);
+    expect(component.container).toHaveTextContent('Welcome to our page!');
+  });
+
+  test('Render Calculator', () => {
+    const component = render(<Calculator />);
+    expect(component.container).toHaveTextContent(`Let's do some Math!`); // eslint-disable-line
```

Comment:
- [ ] Please don't overuse the eslint disable, what issue is eslint giving you guys in here? Try to solve it instead please to comply with Microverse requirements.

Reply:
You're totally right, the changes are done 😄